projects/plugins/boost/app/assets/src/js/pages/benefits/BenefitsInterstitial.svelte

```diff
@@ -8,10 +8,13 @@
 	import ReactComponent from '../../elements/ReactComponent.svelte';
 	import config from '../../stores/config';
 	import Logo from '../../svg/jetpack-green.svg';
+	import { recordBoostEvent } from '../../utils/analytics';
 	import { jetpackURL } from '../../utils/jetpack-url';
 	import { getUpgradeURL } from '../../utils/upgrade';
 
 	function goToCheckout() {
+		const eventProps = {};
+		recordBoostEvent( 'checkout_from_pricing_page_in_plugin', eventProps );
 		window.location.href = getUpgradeURL();
 	}
```

Comment:
The following change should solve the issue:

```suggestion
	async function goToCheckout() {
		const eventProps = {};
		await recordBoostEvent( 'checkout_from_pricing_page_in_plugin', eventProps );
		window.location.href = getUpgradeURL();
	}
```